doc/ref_cert/RC2/chapters/chapter02.rst

```diff
@@ -0,0 +1,611 @@
+Kubernetes Test Cases and Requirements Traceability
+===================================================
+
+.. figure:: ../figures/bogo_lsf.png
```

Comment:
Remove?

Reply:
ok - I do remove it in the orig md first.
trivial